crates/node/src/cli/mod.rs

```diff
@@ -66,14 +73,43 @@ impl RunCmd {
         }
         self.name.as_ref().unwrap()
     }
+}
+
+/// Starknet network types.
+#[derive(Debug, Clone, Copy, clap::ValueEnum, PartialEq)]
+pub enum NetworkType {
```

Comment:
you will need to add a custom network with a user-defined chainid and feeder url to be consistent with the ChainConfig type

Reply:
yepyep coming in another pr:tm:

Reply:
was thinking about it as well, given for app chains it would be much more simpler to have feeder url in the params